language/cc/testdata/includes/BUILD.out

```diff
@@ -0,0 +1,59 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+# gazelle:exclude include
+# gazelle:cc_group unit
+
+#keep
+cc_library(
+    name = "library",
+    hdrs = [
+        "include/lib.h",
+    ],
+    includes = [
+        "include",
+        "other",
+    ],
+)
+
+#keep
+cc_library(
+    name = "library_ext",
+    hdrs = [
+        "include/contrib/lib-ext.h",
+    ],
+    includes = [
+        "include",
+        "include/contrib",
```

Comment:
I think this means the header file can be included as either `include/contrib/lib-ext.h`, `contrib/lib-ext.h`, or `lib-ext.h`. Is that right?